docs/source/validation_rules.rst

```diff
@@ -0,0 +1,714 @@
+================
+Validation Rules
+================
+
+
+.. toctree::
+   :maxdepth: 2
+   :glob:
+
+   *
+
+Overview
+========
+
+When Schematic validates a manifest, it uses a data model. The data model contains a list of validation rules for each component(data-type). This document describes all allowed validation rules currently implemented by Schematic.
+
+Rules that change the validation behavior which must be taken from the following pre-specified list, formatted in the indicated ways, and added to the data model to apply. An `example data model <https://github.com/Sage-Bionetworks/schematic/blob/develop/tests/data/example.model.csv>`_ using each rule is available for reference.
+
+The column the rule refers to must be in the manifest for validation to happen.
+
+Rules can optionally be configured to raise  errors  and prevent manifest submission in the case of invalid entries, or warnings and allow submission when invalid entries are found within the attribute the rule is set for. Validators will be notified of the invalid values in both cases. Default message levels for each rule can be found below under each rule.
+
+Attributes that are not required will raise warnings when invalid entries are identified. For attributes that are not required, if a user does not submit a response, a warning or error will no longer be logged. If you desire raising an error for non-entries, set Required to True in the data model.
+
+Metadata validation is an explicit step in the submission pipeline and must be run either before or during manifest submission.
+
+Validation Types
+================
+
+Validation rules are just one type of validation run by Schematic to ensure that submitted manifests conform to the expectations set by the data model.
+
+This page details how to use Validation Rules, but please refer to `this documentation <https://sagebionetworks.jira.com/wiki/spaces/SCHEM/pages/3302785036>`_ to learn about the other types of validation.
+
```

Comment:
I suggest adding a brief introductory comment here to clarify that some validation rules are implemented in-house, while others use the [Great Expectations](https://greatexpectations.io/) library. This would help readers understand the context and separation of logic.